src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/Types/RoType.cs

```diff
@@ -192,9 +193,7 @@ private RoType[] ComputeInterfaceClosure()
                 }
             }
 
-            // todo: use IEnumerable<T> extension: return ifcs.ToArray()
-            List<RoType> list = new List<RoType>(ifcs);
-            return list.ToArray();
+            return ifcs.ToArray();
```

Comment:
Despite some `using` statements, this library does not have a reference to System.Linq. I guess it is not worth adding one for this. I suggest to remove the `using System.Linq` and instead using code like`
```c#
                    RoType[] arr = new RoType[count];
                    ifcs.CopyTo(arr, 0);
                    return arr;
``